src/services/monitoring.js

```diff
@@ -4,18 +4,31 @@ import {identity, keys, pickBy} from 'lodash';
 
 const API = `${ENDPOINT}api/monitoring/`;
 
+const buildQueryString = filters => keys(
```

Comment:
Extraer a otro utility file. Es el único lugar donde se usa?

Reply:
ya existe otro en utils, este es un poco diferente. En este solo lado se usa.

Reply:
Ese utils dónde está? No es una dependencia?

Reply:
no, https://github.com/indec-it/react-address-commons/tree/develop/src/util

Reply:
Las implementaciones son distintas pero por lo que veo deberían hacer lo mismo.

Reply:
listo